Pegasus/include/pegasus/Math.hpp

```diff
@@ -942,6 +942,30 @@ void CalculateDotProductForeach(
 }
 
 /**
+ * @brief Calculates arbitrary orthonormal vector to the given one
+ * @tparam VectorType glm vector of size 3
+ * @param vector of interest
```

Comment:
param [in]

Reply:
I think we can omit explicit param specifier when it's obvious from the argument type.